wapitiCore/net/intercepting_explorer.py

```diff
@@ -379,7 +379,8 @@ def __init__(
             drop_cookies: bool = False,
             headless: str = "no",
             cookies: Optional[CookieJar] = None,
-            wait_time: float = 2.
+            wait_time: float = 2.,
+            timeout: float = 6.,
```

Comment:
No need for this argument as you are using `crawler_configuration.timeout`

Note that you should be able to directly get the timeout using `self._crawler.timeout`